.github/workflows/test.yml

```diff
@@ -49,7 +50,7 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        php: [8.2, 8.0]
+        php: [8.3, 8.2, 8.0]
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** How about dropping 8.2 version in order to speedup GH checks?

Reply:
Skipping PHP versions might not be the way to go if you want to speed up or reduce the number of CI checks. PHP 8.2 is still widely used and is likely the most essential version to ensure its correct functionality.

In my opinion, a more impactful strategy for reducing the number of checks would be to consider removing the cron job for running daily actions. A commit or merge already trigger all actions to run, which should be sufficient, especially since the test suite on the main branch frequently colours red.

Reply:
Ok, let's test over all supported versions. What I meant is that a contributor should not wait a lot of time to get feedback from the CI. Cron jobs on CI don't block developers in this sense. But I agree about testing using 8.2, of course, it's better to optimize the test suite, but I don't see an easy way of doing it ATM. 

Thank you for your contribution! 💚

Reply:
Ok, let's test over all supported versions. What I meant is that a contributor should not wait a lot of time to get feedback from the CI. Cron jobs on CI don't block developers in this sense. But I agree about testing using 8.2, of course, it's better to optimize the test suite, but I don't see an easy way of doing it ATM.

Thank you for your contribution! 💚